CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [v1.2.3](https://github.com/ably/ably-ruby/tree/v1.2.3)
+
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.2.2...v1.2.3)
+
+**Closed issues:**
+
+- Incorrect ProtocolMessage\#connection\_details object \(overwrites original connection\_details send on CONNECTED state\) [\#377](https://github.com/ably/ably-ruby/issues/377)
```

Comment:
I think this issue needs to be labelled `bug` and so this section should perhaps be entitled 'Fixed Bugs'.

Reply:
done